walkthroughs/week-1-web-development/portfolio-walkthrough.md

```diff
@@ -461,12 +461,12 @@ If you have extra time left over, consider adding a page that describes your
 experiences with SPS so far, or writing JavaScript that makes your page
 interactive, or using CSS to make your page more beautiful.
 
-Next week, you'll learn about server-side code and add a comment feature to your
-portfolio.
+Next week, you'll learn about server-side code and add a "contact me" feature to
```

Comment:
Did you mean to add this file to the pull request?

Reply:
Once again, I did not mean to add this file must have been an accident. Sorry about that I'll be more vigilant about which files are included in the future.